packages/components/icon/package.json

```diff
@@ -49,5 +53,8 @@
   "publishConfig": {
     "registry": "https://npm.pkg.github.com/",
     "access": "public"
+  },
+  "engines": {
+    "node": ">=16"
```

Comment:
Why is this required? It's a frontend package.

Reply:
It isn't, to be removed.